homeassistant/components/mqtt/light/schema_basic.py

```diff
@@ -182,6 +185,7 @@
             vol.Optional(CONF_COLOR_TEMP_COMMAND_TOPIC): valid_publish_topic,
             vol.Optional(CONF_COLOR_TEMP_STATE_TOPIC): valid_subscribe_topic,
             vol.Optional(CONF_COLOR_TEMP_VALUE_TEMPLATE): cv.template,
+            vol.Optional(CONF_COLOR_TEMP_KELVIN, default=False): cv.boolean,
```

Comment:
Will we do some deprecation where the default is changed to `True` in the future?

Reply:
There is no deprecation plan yet, but first I would like to see if we need one. If we are about to deprecate, we can start logging mireds are used, or pop up a repair issue for manual configured items for about a year. For now it seems okay to just add the option.

Reply:
OK, sounds good.